template-compose/app/src/main/java/co/nimblehq/template/compose/ui/screens/home/HomeScreen.kt

```diff
@@ -22,6 +25,10 @@ fun HomeScreen(
     viewModel: HomeViewModel = hiltViewModel(),
     navigator: (destination: AppDestination) -> Unit
 ) {
+    val context = LocalContext.current
+    viewModel.error.collectAsEffect { e -> e.showToast(context) }
```

Comment:
Any reason we don't make use of the `Toaster` class here? 🤔

Reply:
@ryan-conway ComposableScreen is not the same as a Fragment so we can easily bind a `Toaster` component. Let me check whether we can do anything to make use of it. 🤔

Reply:
@ryan-conway I tried to make it work with the ComposableScreen. However, from the updated code, I don't see much benefit of having a Toaster component with Jetpack Compose anymore 🤷‍♂️ You can see my insights from the diff below:

![image](https://user-images.githubusercontent.com/16315358/231944687-9816ee37-88c4-4c80-9519-dada61c144a8.png)

![image](https://user-images.githubusercontent.com/16315358/231944961-3168f367-0405-4ab7-bb67-af20b57d906a.png)

Instead of having a Toaster component, we can simply have a Context extension to show the Toast 💡

Reply:
@ryan-conway Fixed in https://github.com/nimblehq/android-templates/pull/449/commits/82c99e598454daeaab58d3916514ada106a9ade5 👉 Replace the simple `Toaster` component by an extension.